pandas/core/algorithms.py

```diff
@@ -1491,6 +1491,8 @@ def take_nd(arr, indexer, axis=0, out=None, fill_value=np.nan, mask_info=None,
     if is_sparse(arr):
         arr = arr.get_values()
 
+    arr = np.array(arr, copy=False)
```

Comment:
``np.asarray`` slightly more idiomatic

Reply:
Ah, yes (for some reason I wanted the `copy=False` keyword, but for `asarray` that is of course just the default behaviour)